src/main/scala/io/shiftleft/fuzzyc2cpg/astnew/AstToCpgConverter.scala

```diff
@@ -827,6 +843,13 @@ class AstToCpgConverter[NodeBuilderType, NodeType, EdgeBuilderType, EdgeType](
     cpgNode
   }
 
+  private def acceptChildren(node: AstNode, withArgEdges: Boolean = false): Unit = {
+    node.getChildIterator.forEachRemaining { child =>
+      if (withArgEdges) context.addArgumentEdgeOnNextAstEdge = true
```

Comment:
I guess `context.addArgumentEdgeOnNextAstEdge = withArgEdges` is what you really want. Not sure though.

Reply:
Good point! Some coffee is in order :grin: